assignment/index.html

```diff
@@ -62,10 +62,35 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
+      var header = [Object.keys(healthCenters[0])];
+      var TempArr = [Object.values(healthCenters)];
+      healthCentersArr = header.push(TempArr);
+      for (i=0; i < healthCentersArr.length; i++){console.log(healthCentersArr)};
```

Comment:
You are extremely close. But haven't quite gotten it. 

`header` is an array that contains two sub arrays. The first is correct, but the second `TempArr` is an array with a collection of objects. 

Do you see how these results looks different in the console:

![image](https://user-images.githubusercontent.com/8103418/52538544-f9e26a00-2d41-11e9-81c3-d6ae287c6832.png)

versus
![image](https://user-images.githubusercontent.com/8103418/52538536-ef27d500-2d41-11e9-8916-0ca020a1e328.png)



![image](https://user-images.githubusercontent.com/8103418/52538451-be936b80-2d40-11e9-8749-002dc6b461e8.png)